hack/jenkins/common.sh

```diff
@@ -80,10 +80,12 @@ export MINIKUBE_BIN="out/minikube-${OS_ARCH}"
 export E2E_BIN="out/e2e-${OS_ARCH}"
 chmod +x "${MINIKUBE_BIN}" "${E2E_BIN}" out/docker-machine-driver-*
 
-
-if pgrep "minikube|e2e-"; then
-  echo "WARNING: other instances of minikube may be running:"
-  ps -afe | egrep "minikube|e2e-"
+procs=$(pgrep "minikube-${OS_ARCH}|e2e-${OS_ARCH}" || true)
+if [[ "${procs}" != "" ]]; then
+  echo "ERROR: found stale test processes to kill:"
+  ps -f -p ${procs} || true
+  kill ${procs} || true
+  kill -9 ${procs} || true
```

Comment:
why not just -9 only? the second command will kill slow graceful shutdowns anyway, otherwise it will just create unnecessary noise as most of the processes will not be found to be killed.

Reply:
I give programs the chance to cleanup whenever possibly, mostly to avoid corrupted writes. 

Even if a -9 is just a fork and exec away, you can't say I didn't warn the processes first!